tf_encrypted/keras/activations.py

```diff
@@ -0,0 +1,39 @@
+"""Provide activation functions"""
+from tf_encrypted.protocol.protocol import get_protocol
```

Comment:
Once the APIs settle a bit it will probably be best to *not* use relative imports but do what you're doing here instead -- creates better modularity. So perhaps stay away from relative imports here 👍

Reply:
I like this one `from tf_encrypted import get_protocol`

Reply:
good point.